python/tk_multi_publish2/api/plugins/publish_plugin_instance.py

```diff
@@ -8,18 +8,13 @@
 # agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
 # not expressly granted therein are reserved by Shotgun Software Inc.
 
-from contextlib import contextmanager
 import traceback
 
-try:
-    # In Python 3 the interface changed, getargspec is now deprecated.
-    # getfullargspec was deprecated in 3.5, but that was reversed in 3.6.
-    from inspect import getfullargspec as getargspec
-except ImportError:
-    # Fallback for Python 2
-    from inspect import getargspec
-
 import sgtk
+
+from contextlib import contextmanager
+from inspect import getfullargspec
```

Comment:
Can you keep those standard lib imports before the local ones (sgtk) please?

Reply:
@eduardoChaucaGallegos quick suggestion: use `isort`: https://pypi.org/project/isort/

Reply:
Still something here. Why is traceback alone and no sorted?